webapp-django/crashstats/crashstats/static/crashstats/css/daily.less

```diff
@@ -5,6 +5,10 @@ h1 {
     font-size: 1.90em;
     font-weight: bold;
 }
+.page-heading p.new-daily-report-link {
+    float: right;
+}
+
```

Comment:
As I said, I do not mind at all going your way. You have strong arguments about it. The only thing is, there are 6 places where we have that kind of links, not 4: TCBS old and new, Daily Crashes old and new, and report/list/ and Signature Report.